docs/questions/query-builder/expressions/now.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Now
+---
+
+# Now
```

Comment:
@nllho Unfortunately, `now` has been pushed to the next release. I would excise this content from this PR, and ship it (without backporting it) in a separate PR.